test/e2e/run-api-specs-multichain.ts

```diff
@@ -111,9 +112,10 @@ async function main() {
       extensionId: string;
     }) => {
       await unlockWallet(driver);
-
-      // Navigate to extension home screen
-      await driver.navigate(PAGES.HOME);
+      const homePage = new HomePage(driver);
+      await homePage.checkPageIsLoaded();
+      // We don't have balance so we expect to see Fund Your Wallet
+      await homePage.checkExpectedBalanceIsDisplayed('0', 'ETH');
```

Comment:
in the api specs we do expect the fund your wallet banner, that's why the test was flaky